ignite/services/network/mocks/bank_client.go

```diff
@@ -16,6 +16,11 @@ type BankClient struct {
 	mock.Mock
 }
 
+func (_m *BankClient) DenomOwners(ctx context.Context, in *types.QueryDenomOwnersRequest, opts ...grpc.CallOption) (*types.QueryDenomOwnersResponse, error) {
+	//TODO implement me
+	panic("implement me")
+}
+
```

Comment:
This [comment](https://github.com/ignite/cli/pull/2675#issuecomment-1198991030) might help to auto generate the mock